languages/ocaml/generic/ml_to_generic.ml

```diff
@@ -181,7 +181,8 @@ and stmt e : G.stmt =
       let var = { G.vinit = Some v2; vtype = None } in
       let n = G.N (G.Id (v1, G.empty_id_info ())) |> G.e in
       let next =
-        G.AssignOp (n, (nextop, tok), G.L (G.Int (Some 1, tok)) |> G.e) |> G.e
+        G.AssignOp (n, (nextop, tok), G.L (G.Int (Parsed_int.of_int 1)) |> G.e)
```

Comment:
we're losing the tok here.

Reply:
we were, but this actually was a token which wasn't fair to associate to the `1` anyways -- it came from somewhere else. ive added it back, but personally i might prefer that we don't associate tokens to semantic structures that don't correspond to them at all